arrow-data-source/standard/src/main/scala/com/intel/oap/spark/sql/execution/datasources/arrow/ArrowFileFormat.scala

```diff
@@ -94,6 +94,10 @@ class ArrowFileFormat extends FileFormat with DataSourceRegister with Serializab
           override def close(): Unit = {
             writeQueue.close()
           }
+
+          override def path(): String = {
```

Comment:
Fixed through shim layer.
TODO: it seems we can put the new interface method here and omit override key word.